Formula/gz-launch7.rb

```diff
@@ -2,7 +2,7 @@ class GzLaunch7 < Formula
   desc "Launch libraries for robotics applications"
   homepage "https://github.com/gazebosim/gz-launch"
   url "https://github.com/gazebosim/gz-launch.git", branch: "main"
-  version "7.999.999~0~20221117"
+  version "6.999.999~0~20230728"
```

Comment:
@scpeters would this cause problems? I think 6.999.999 is the correct value, but the old one had 7.999.999.

Reply:
yeah, it should be fine since there are no bottles built for this yet